README.md

```diff
@@ -9,6 +9,7 @@ Container images for [Samba](https://samba.org) services.
 * Are used by the [samba-operator](https://github.com/samba-in-kubernetes/samba-operator) for Kubernetes
 * Don't require Kubernetes
 * Are [available at quay.io](https://quay.io/organization/samba.org)
+* Are used by the [ceph project](https://github.com/ceph/ceph) to provide  SMB services on CephFS.
```

Comment:
this one still has a trailing period at the end of the line

Reply:
done. removed